libs/remix-ui/editor/src/lib/providers/codeActionProvider.ts

```diff
@@ -134,4 +104,35 @@ export class RemixCodeActionProvider implements monaco.languages.CodeActionProvi
       isPreferred: true
     })
   }
+
+  async fnWithParamsQFMsg(model, paramNodes, fix, error, isOldAST) {
```

Comment:
Is it possible to have some basic typing of the parameters being passed in? Typing also serves as documentation when going over the code in the future.

Reply:
I will try to add more documentation and type definitions

Reply:
done